tests/playwright/sanity/modules/floating-buttons/floating-elements-admin.test.ts

```diff
@@ -0,0 +1,182 @@
+import { parallelTest as test } from '../../../parallelTest';
+import FloatingElementPage from './floating-element-page';
+import { expect } from '@playwright/test';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Verify floating buttons editor, admin page and front page behavior', () => {
+	test.afterEach( async ( { browser, apiRequests }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		await floatingElPage.deleteAllFloatingButtons();
+		await page.close();
+	} );
+
+	test( 'Verify editor behavior by creating post through API and also FE behavior', async (
+		{
+			browser,
+			page,
+			apiRequests,
+		}, testInfo ) => {
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		const editor = await floatingElPage.createFloatingButtonWithAPI();
+		await floatingElPage.waitForPanel();
+		const navigatorId = 'header [aria-label="Structure"]';
+		const navigator = page.locator( navigatorId );
+		// We need to check that the normal elements are not present, that the navigator is visible and the title is correct.
+		await expect( navigator ).toBeHidden();
+		const addButton = page.locator( 'header [aria-label="Add Element"]' );
+		await expect( addButton ).toBeHidden();
+		const panelTitle = page.locator( '#elementor-panel-header-title' );
+		await expect( panelTitle ).toHaveText( 'Edit Single Chat' );
+
+		// Check that we are seeing our custom advanced tab and that the normal advanced tab is hidden.
+		const navigationContainer = page.locator( '.elementor-panel-navigation' );
+		const buttons = navigationContainer.locator( 'button' );
+		const visibleButtons = buttons.filter( { has: page.locator( ':visible' ) } );
+		await expect( visibleButtons ).toHaveCount( 3 );
+		const advancedSettingsCustom = page.locator( '.elementor-tab-control-advanced-tab-floating-buttons' );
+		await expect( advancedSettingsCustom ).toBeVisible();
+		await advancedSettingsCustom.click();
```

Comment:
It is better to avoid any actions after the assert. It is better to perform a new action in another step.